mcy/src/components/Layout/Header.jsx

```diff
@@ -1,5 +1,31 @@
+import { Stack, styled, Typography, IconButton } from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+
 const Header = () => {
-  return <div>Header</div>;
+  return (
+    <>
+      <HeaderWrapper
+        borderBottom="1px solid #000"
+        justifyContent="center"
+        padding="5px"
```

Comment:
```suggestion
        borderBottom="1px solid #000"
        justifyContent="center"
        padding="5px"
```
이건 어짜피 styled 할거면 아래 코드에 다가 몰아주는게 더 깔끔할 듯

```
const HeaderWrapper = styled(Stack)`
  border-bottom=1px solid #000;
  justify-content=center;
  padding=5px;
  flex-direction: row;
  align-items: center;
  width: 100vw;
  position: fixed;
  top: 0;
`;
```